pkg/template/template_test.go

```diff
@@ -116,7 +116,7 @@ func TestParameterGenerators(t *testing.T) {
 	for i, test := range tests {
 		processor := NewProcessor(test.generators)
 		template := api.Template{Parameters: []api.Parameter{test.parameter}}
-		err, _ := processor.GenerateParameterValues(&template)
+		err := processor.GenerateParameterValues(&template)
```

Comment:
it's not that interesting but it's somewhat interesting, particularly for if/when something gets refactored/restructured.